### ISSUE TITLE
Some issues and suggestions

### ISSUE THREAD

**Issue by Krouzu** [Jire/Charlatano#56]
**Issues:**
1. When aiming with pistol and aim key held down, when shot in the head the aimbot jerks down.
2. (Might not be an issue) When you shoot the deagle it goes down and comes back up making it hard to shoot a second time accurately.

**Suggestions:** 
1. Ability to change aim bone
2. Triggerbot key
3. Wall hack toggle key

**Comment by SquareHeadd** [comment]
I agree, to be able to toggle on/off would be great, I have the same thing with deagle
